docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md

```diff
@@ -42,8 +47,20 @@ In addition to the default boilerplate, we can add additional entries to the hos
 
 {% include code.html language="yaml" file="hostaliases-pod.yaml" ghlink="/docs/concepts/services-networking/hostaliases-pod.yaml" %}
 
-The hosts file content would look like this:
+This Pod can be started with the following commands:
+
+```shell
+$ kubectl apply -f hostaliases-pod.yaml
+pod "hostaliases-pod" created
+
+$ kubectl get pod -a
```

Comment:
can you do `-o=wide` here so it prints out the pod IP. that way the IP of this entry

```
10.244.3.80	hostaliases-pod
```

becomes clearer

Reply:
@rickypai Okay!